docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -16,9 +16,26 @@ NuGet's behavior is driven by the accumulated settings in one or more `NuGet.Con
 | Scope | `NuGet.Config` file location | Description |
 | --- | --- | --- |
 | Solution | Current folder (aka Solution folder) or any folder up to the drive root.| In a solution folder, settings apply to all projects in subfolders. Note that if a config file is placed in a project folder, it has no effect on that project. |
-| User | **Windows:** `%appdata%\NuGet\NuGet.Config`<br/>**Mac/Linux:** `~/.config/NuGet/NuGet.Config` or `~/.nuget/NuGet/NuGet.Config` (varies by OS distribution) <br/>Additional configs are supported on all platforms. These configs cannot be edited by the tooling. </br> **Windows:** `%appdata%\NuGet\config\*.Config` <br/>**Mac/Linux:** `~/.config/NuGet/config/*.config` or `~/.nuget/config/*.config` | Settings apply to all operations, but are overridden by any project-level settings. |
+| User | **Windows:** `%appdata%\NuGet\NuGet.Config`<br/>**Mac/Linux:** `~/.config/NuGet/NuGet.Config` or `~/.nuget/NuGet/NuGet.Config` (varies by tooling) <br/>Additional configs are supported on all platforms. These configs cannot be edited by the tooling. </br> **Windows:** `%appdata%\NuGet\config\*.Config` <br/>**Mac/Linux:** `~/.config/NuGet/config/*.config` or `~/.nuget/config/*.config` | Settings apply to all operations, but are overridden by any project-level settings. |
 | Computer | **Windows:** `%ProgramFiles(x86)%\NuGet\Config`<br/>**Mac/Linux:** `$XDG_DATA_HOME`. If `$XDG_DATA_HOME` is null or empty, `~/.local/share` or `/usr/local/share` will be used (varies by OS distribution)  | Settings apply to all operations on the computer, but are overridden by any user- or project-level settings. |
 
+> [!Note]
+> On Mac/Linux, the user config file location varies by tooling. Dotnet CLI uses `~/.nuget/NuGet` folder, while Mono uses `~/.config/NuGet` folder. 
+
+### On Mac/Linux, the user-level config file location varies by tooling. 
+On Mac/Linux, the user config file location varies by tooling. 
```

Comment:
```suggestion
```

Reply:
Thanks for reviewing! Is there a way to reference this part if it's not a heading level? As NuGet.exe command needs to add a reference to this part.

Reply:
the suggested change is not on the heading, it's on the line after the heading that repeats what the heading already says

Reply:
The header is great and I just removed the duplicate line.